utils/build-script-impl

```diff
@@ -3015,6 +3016,25 @@ for host in "${ALL_HOSTS[@]}"; do
     done
 done
 
+function printJSONTimestamp {
```

Comment:
Any reason to prefer the POSIX style definition rather than bash?

```bash
printJSONTimestamp() {
 ...
}
```

and similar for the other functions.

Reply:
Good catch, missed checking the style of the other functions -- should be matched in 17388de4048